docs/components/concepts/clusters.md

```diff
@@ -53,6 +53,36 @@ Additionally in the Starter Plan, the following applies to **development cluster
 - **Cluster is not highly available & includes less hardware**: Reduced hardware resources and availability compared to production cluster (for example, one Zeebe node only).
 - **Shorter history of processes and decisions**: Data retention in Operate, Optimize, and Tasklist is reduced to one day. For example, pending or historical process instances are deleted after one day as per the [fair usage limits of the Starter plan](https://camunda.com/legal/fair-usage-limits-for-starter-plan/).
 
-:::caution
-**Cluster auto-pause** is not yet available and only applies to non-Enterprise clusters. Development clusters will be paused if they go unused for two hours. When a cluster is paused, not all functionality will be limited, including the execution of BPMN timers and BPMN message catch events.
+## Autopause
+
+Free and Free Trial plan `dev` (or untagged) clusters are automatically paused 8 hours after the creation of the cluster or since you last resumed it from a paused state.
+
+No data is lost while a cluster is paused. All execution and configuration is saved, but cluster components such as Zeebe and Operate are temporarily disabled until you resume the cluster.
+
+- Production clusters and clusters tagged as `test`, `stage`, or `prod` are exempt and do not autopause.
+- Starter, Professional, and Enterprise customer clusters are exempt and do not autopause.
+- Paused clusters are automatically deleted after 30 consecutive days in a paused state. You can change the tag to avoid cluster deletion.
+- Autopause occurs regardless of cluster usage.
+- You can resume a paused cluster at any time. See [Resume your cluster](/components/console/manage-clusters/resume-cluster.md/).
+
+The following table shows the cluster autopause schedule for each plan and tag.
+
+|              | Untagged or `dev`\* | `test`   | `stage`  | `prod`   |
+| ------------ | ------------------- | -------- | -------- | -------- |
+| Free Trial   | 8 hours             | _exempt_ | _exempt_ | _exempt_ |
+| Free         | 8 hours             | _exempt_ | _exempt_ | _exempt_ |
+| Starter      | _exempt_            | _exempt_ | _exempt_ | _exempt_ |
+| Professional | _exempt_            | _exempt_ | _exempt_ | _exempt_ |
+| Enterprise   | _exempt_            | _exempt_ | _exempt_ | _exempt_ |
+
+\* Untagged clusters are treated as `dev` tagged clusters.
+
+:::tip
+
+To avoid having to keep resuming your cluster(s) after an autopause, you can:
```

Comment:
i'd say it is more about avoiding that my cluster falls asleep ;-)

Reply:
Thanks @ultraschuppi, agreed about reframing it - I have changed that line to "To avoid your clusters being automatically paused, you can:"